curriculum/challenges/english/blocks/workshop-quincys-job-tips/68eaa3593b6ffd019d772a1f.md

```diff
@@ -17,11 +19,15 @@ You should have a `p` element below the `blockquote` element.
 assert.exists(document.querySelector('section:first-of-type > blockquote + p'));
 ```
 
-Your `p` element should have the text `—Quincy Larson, How to Learn to Code and Get a Developer Job [Full Book]`.
+Your `p` element should have the text `&mdash;Quincy Larson, How to Learn to Code and Get a Developer Job [Full Book]`.
 
 ```js
 const pEl = document.querySelector('section:first-of-type > blockquote + p');
-assert.equal(pEl?.innerText.trim(), '—Quincy Larson, How to Learn to Code and Get a Developer Job [Full Book]');
+const html = pEl?.innerHTML.trim();
+assert(
+  html ==='—Quincy Larson, How to Learn to Code and Get a Developer Job [Full Book]' ||
+  html === '&mdash;Quincy Larson, How to Learn to Code and Get a Developer Job [Full Book]' 
+);
```

Comment:
if you are allowing both, use the [`oneOf` assert method](https://www.chaijs.com/api/assert/#method_oneof)